packages/blade/src/components/Chip/Chip.stories.tsx

```diff
@@ -0,0 +1,291 @@
+import type { ComponentStory, Meta } from '@storybook/react';
+import React from 'react';
+import { Code, Text, Title } from '../Typography';
+import type { ChipGroupProps } from './ChipGroup';
+import { ChipGroup as ChipGroupComponent } from './ChipGroup';
+import { Chip as ChipComponent } from './Chip';
+import {
+  PaymentLinksIcon,
+  TagIcon,
+  SmartphoneIcon,
+  ThumbsUpIcon,
+  ThumbsDownIcon,
+} from '~components/Icons';
+import { Sandbox } from '~utils/storybook/Sandbox';
+import StoryPageWrapper from '~utils/storybook/StoryPageWrapper';
+import type { BladeElementRef } from '~utils/types';
+import { Button } from '~components/Button';
+import { Box } from '~components/Box';
+import { getStyledPropsArgTypes } from '~components/Box/BaseBox/storybookArgTypes';
+import iconMap from '~components/Icons/iconMap';
+
+const Page = (): React.ReactElement => {
+  return (
```

Comment:
remove figma+sandpack example for chip and put an alert which says Chips cannot be used individually and shall only be used with `ChipGroup` and provide a link to the ChipGroup Stories. Also, add the description that this story is only meant for demonstrating props of chip component.